### PR TITLE
feat(nimbus): add overview form to new nimbus ui

### LINKED ISSUE #10841: Overview Form
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4617)

### PR DESCRIPTION
**Pull request by jaredlockhart**
Becuase
    
* We need to add the overview to the new Nimbus UI
    
This commit
    
* Adds the overview form
* Adds documentation link add/remove functionality with HTMX
    
fixes #10841
image

image

### CODE REVIEW

**Comment by jaredlockhart** [comment]
Blocked on https://github.com/mozilla/experimenter/pull/11925

File: experimenter/experimenter/nimbus_ui_new/views.py [hunk 1]
```diff
@@ -34,6 +37,19 @@ def get_form_kwargs(self):
...
+        super().form_valid(form)
+        return self.render_to_response(self.get_context_data(form=form))
+
+
+class RenderParentResponseMixin:
+    def form_valid(self, form):
+        super().form_valid(form)
+        form = super().form_class(instance=self.object)
+        return self.render_to_response(self.get_context_data(form=form))
+
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus_ui_new/views.py:51]: I Liked this mixin idea

File: experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/edit_overview.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            </label>
+          </div>
+          <div class="col-2 text-end">{{ form.risk_revenue }}</div>
+        </div>
+        <div class="row mb-3">
+          <div class="col-10">
+            <label for="id_risk_partner_related" class="form-label">
+              Does this experiment impact or rely on a partner or outside company (e.g. Google, Amazon) or deliver any encryption or VPN?
+              <a target="_blank"
+                 href="https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#PrefFlipandAddOnExperiments-Isthisstudypartnerrelated?riskPARTNER">
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/edit_overview.html:105]: for links like these, in my current work I am making the constant dict and will use it here, I can replace all that later, just wanted to let you know

**Review by yashikakhurana — APPROVED** [review]
Tested it locally, works smoothly, thanks @jaredlockhart 🎉

**Pull request merged**